dask/base.py

```diff
@@ -61,8 +62,40 @@ def visualize(self, filename='mydask', format=None, optimize_graph=False,
         return visualize(self, filename=filename, format=format,
                          optimize_graph=optimize_graph, **kwargs)
 
+    def persist(self, **kwargs):
+        """ Persist this dask collection into memory
+
+        This turns a lazy Dask collection into a Dask collection with the same
+        metadata, but now with its results actively computing.  For example a
+        lazy dask.array built up from many lazy calls will now be a dask.array
+        of the same shape, dtype, etc., but now with all of those lazy
+        operations either in memory (for single-machine computing) or actively
```

Comment:
"lazy operations in memory" doesn't really make sense. Maybe talk about "the result of those lazy operations", or just say the array is stored in memory as a NumPy array?